pkg/kubelet/kubelet.go

```diff
@@ -2186,18 +2192,19 @@ func (kl *Kubelet) updateRuntimeUp() {
 
 	s, err := kl.containerRuntime.Status()
 	if err != nil {
-		klog.Errorf("Container runtime sanity check failed: %v", err)
+		klog.ErrorS(err, "Container runtime sanity check failed")
 		return
 	}
 	if s == nil {
-		klog.Errorf("Container runtime status is nil")
+		klog.ErrorS(nil, "Container runtime status is nil")
 		return
 	}
 	// Periodically log the whole runtime status for debugging.
-	klog.V(4).Infof("Container runtime status: %v", s)
+	klog.V(4).InfoS("Container runtime status", "status", s)
 	networkReady := s.GetRuntimeCondition(kubecontainer.NetworkReady)
 	if networkReady == nil || !networkReady.Status {
-		klog.Errorf("Container runtime network not ready: %v", networkReady)
+		err := fmt.Errorf(networkReady.String())
+		klog.ErrorS(err, "Container runtime network not ready")
```

Comment:
is there a recommended way to transform a non-ready condition into the error?

Reply:
I think in this case I'd instead do `nil` error and a keypair `networkReady=networkReady.String()`

/cc @serathius

Reply:
sg